product_delivery_destination/models/product_template.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    delivery_prefecture = fields.Many2one('res.country.state', string="Delivery Prefecture", domain="[('is_deliverable','=', True)]")
+    delivery_city = fields.Many2one('delivery.city', string="Delivery City")
```

Comment:
To follow the standard naming rule.

```suggestion
    delivery_state_id = fields.Many2one('res.country.state', string="Delivery Prefecture", domain="[('is_deliverable','=', True)]")
    delivery_city_id = fields.Many2one('delivery.city', string="Delivery City")
```